pkg/k8s/labels.go

```diff
@@ -144,14 +144,18 @@ const (
 	// disableExternalProfilesAnnotation config.
 	ProxyDisableExternalProfilesAnnotation = ProxyConfigAnnotationsPrefix + "/disable-external-profiles"
 
+	// IdentityModeDefault is assigned to IdentityModeAnnotation to
+	// use the control plane's default identity scheme.
+	IdentityModeDefault = "default"
+
 	// IdentityModeDisabled is assigned to IdentityModeAnnotation to
 	// disable the proxy from participating in automatic identity.
 	IdentityModeDisabled = "disabled"
 
 	// IdentityModeOptional is assigned to IdentityModeAnnotation to
 	// optionally configure the proxy to participate in automatic identity.
 	//
-	// Will be deprecated soon.
+	// Deprecated.
```

Comment:
The convention for this is `Deprecated: <reason>`. See [here](https://github.com/golang/go/wiki/Deprecated).

Reply:
Actually, just going to delete this, as there's no reason to use it going forward.